package.json

```diff
@@ -50,7 +50,7 @@
     "chai": "^4.0.1",
     "enzyme": "^2.3.0 || ^3.0.0",
     "enzyme-adapter-react-16": "^1.0.0",
-    "jest": "^20.0.0",
+    "jest": "^21.0.1",
```

Comment:
can be locked at exact version `21.0.1`?

Reply:
That's up to you, if it's locked then you can be sure that jest version differences aren't causing problems in the tests.

Reply:
Sure I would recommend locking it at exact version.